test/enrollment_repository_test.rb

```diff
@@ -1,77 +1,91 @@
 require "minitest/autorun"
 require "minitest/pride"
 require "enrollment_repository"
-require "pry"
-require "enrollment"
 
 class EnrollmentRepositoryTest < Minitest::Test
-  def fixture_path  
-    File.expand_path("fixtures/Kindergartners in full-day program.csv", __dir__)
+  def kindergarten_participation_access_hash
```

Comment:
This function was renamed for clarity and created in the hashed attributename => file_path convention used earlier instead of the keys passed in to and translated by the enrollment_repository class.